File: src/math/NumberGuss.kt
package math

fun main(args: Array<String>) {

    val times = readLine()?.toInt() ?: 0

    for (i in 0 until times){
        var sum = 0
        val number = readLine()?.toInt() ?: 0
        for (num in 1 until number){
            if (number % num == 0){
                sum += num
            }
        }
        println(sum)
    }


}

File: src/math/numberGuss.dart
import 'dart:io';

void main(){

	int times = int.parse(stdin.readLineSync());

	for (var i = 0; i < times; i++) {
		var sum = 0;
		int number = int.parse(stdin.readLineSync());

		int j = 1;
		while(number > j){
			if (number % j == 0){
				sum += j;
			}

			j++;
		}

		print(sum);
	}

}